### ISSUE TITLE
Docker compose failed

### ISSUE THREAD

**Issue by GalochkinKirov** [smi2/tabix.ui#9]
Cannot build project in **docker compose**.

> bower d3                                          resolution Unsuitable resolution declared for d3: ~4.4.1
> bower                                              ECONFLICT Unable to find suitable version for d3
> ERROR: Service 'api' failed to build: The command '/bin/sh -c npm install &&     bower install --allow-root &&     gulp build' returned a non-zero code: 1

**Comment by bessarabov** [comment]
`docker build` is failing:

```

$ git show|grep commit
commit 7e1af85683b0fc550e9dec43efb1a6f13751953b
$ docker build --tag tabix.ui .
...
bower pikaday#^1.4.0                                 extract archive.tar.gz
bower pikaday#^1.4.0                                resolved https://github.com/dbushell/Pikaday.git#1.5.1
bower numbro#^1.8.0                                  extract archive.tar.gz
bower numbro#^1.8.0                                 resolved https://github.com/foretagsplatsen/numbro.git#1.10.1
bower moment#^2.13.0                                 extract archive.tar.gz
bower echarts#^3.5.2                                resolved https://github.com/ecomfe/echarts.git#3.5.3
bower moment#^2.13.0                                resolved https://github.com/moment/moment.git#2.18.1
bower ace-builds#^1                                 mismatch Version declared in the json (1.2.3) is different than the resolved one (1.2.6)
bower ace-builds#^1                                 resolved https://github.com/ajaxorg/ace-builds.git#1.2.6

Please note that,
    angular-resizable#1.2.0 depends on angular#~1.3.15 which resolved to angular#1.3.20
    angular-animate#1.5.9, angular-aria#1.5.9 depends on angular#1.5.9 which resolved to angular#1.5.9
    angular-translate#2.15.1, angular-translate#2.15.1 depends on angular#>=1.2.26 <1.7 which resolved to angular#1.5.9
    ng-csv#0.3.6 depends on angular#~1 which resolved to angular#1.5.9
    angular-local-storage#0.2.7 depends on angular#^1.x which resolved to angular#1.5.9
    angular-ui-grid#4.0.4 depends on angular#>=1.4.0 1.5.x which resolved to angular#1.5.9
    angular-ui-ace#0.2.3 depends on angular#~1.x which resolved to angular#1.5.9
    fun-metis-menu#60f4959894 depends on angular#>=1.2 which resolved to angular#1.5.9
    angular-dragdrop#1.0.13 depends on angular#* which resolved to angular#1.5.9
    ng-scrollbars#0.0.11 depends on angular#^1.2.21 which resolved to angular#1.5.9
    ngHandsontable#0.13.0 depends on angular#~1.5.0 which resolved to angular#1.5.9
    angular-gridster#0.13.15 depends on angular#>= 1.2.0 which resolved to angular#1.5.9
    angular-material#1.1.4 depends on angular#^1.4.8 which resolved to angular#1.5.9
    angular-ui-router#0.3.2 depends on angular#^1.0.8 which resolved to angular#1.5.9
    angular-mocks#1.5.11, Tabix depends on angular#1.5.11 which resolved to angular#1.5.11
    angular-messages#1.6.2, angular-sanitize#1.6.2 depends on angular#1.6.2 which resolved to angular#1.6.2
Resort to using angular#~1.5.11 which resolved to angular#1.5.11
Code incompatibilities may occur.

bower d3                                          resolution Unsuitable resolution declared for d3: ~4.4.1
bower                                              ECONFLICT Unable to find suitable version for d3
The command '/bin/sh -c npm install &&     bower install --allow-root &&     gulp build' returned a non-zero code: 1
$
```

**Comment by isublimity** [comment]
try new release
